rllib/agents/trainer.py

```diff
@@ -678,13 +678,6 @@ def _evaluate(self):
         Note that this default implementation does not do anything beyond
         merging evaluation_config with the normal trainer config.
         """
-        if not self.config["evaluation_config"]:
```

Comment:
Why remove this warning?

Reply:
Ah, yes, if only `evaluation_interval` is changed (w/o adding `evaluation_config`), which I think would be a valid config, then this check will error.